test/new-e2e/tests/containers/k8s_test.go

```diff
@@ -542,6 +542,41 @@ func (suite *k8sSuite) TestNginx() {
 		},
 	})
 
+	suite.testMetric(&testMetricArgs{
+		Filter: testMetricFilterArgs{
+			Name: "nginx.net.connections",
```

Comment:
If DataDog/test-infra-definitions#1246 had an impact on the tags cardinality of the `nginx` check, it should already be visible on the metric assertion for `nginx.net.request_per_s` done above, at line 491.
This new assertion doesn’t increase the test coverage as we only see that, for the same check on the same pods, we get exactly the same tag set.